docs/products/clickhouse/howto/integrate-pg.rst

```diff
@@ -0,0 +1,106 @@
+``PG_SERVICE_NAME``              Name of the PostgreSQL service you use for the integration.
+``PG_DATABASE``                  Name of PostgreSQL database you're integrating.
+``PG_SCHEMA``                    Name of PostgreSQL schema.
+``PG_TABLE``                     Name of the PostgreSQL table you use for the integration.
+============================     ==========================================================================================================
+
+Create an integration
+----------------------
+
+You can create an integration with the help of :ref:`Aiven CLI <avn_service_integration_create>`, or through the `Aiven console <https://console.aiven.io/>`_ by following these steps:
+
+1. Log in to the `Aiven web console <https://console.aiven.io/>`_, choose the right project, and select your Aiven for ClickHouse service.
+#. In the *Overview* tab, scroll till the section *Service integrations* and click on **Set up integration**. A modal window with available integration options will appear.
+#. Select PostgreSQL from the list of the options.
+#. Choose your service from the list. This is also a place where you can trigger creation of a new Aiven for PostgreSQL service.
+#. Click on **Enable**.
+#. Your PostgreSQL service will be added to the list of enabled service integrations.
+#. You can now close *Service integrations* modal window.
+#. The PostgreSQL integration database will now be added.
+
+The newly created database name has the following format: ``service_PG_SERVICE_NAME_PG_DATABASE_PG_SCHEMA``, for example, ``service_myPGService_myDatabase_mySchema``.
+
+.. note::
+
+    When connecting to an Aiven for PostgreSQL, we connect as the main service user of that service, which has access to all the PostgreSQL tables. SELECT and INSERT privileges are granted to the main service user. It is up to the main service user to grant access to other users.
+
+
+Update PostgreSQL integration settings
+-----------------------------------------
+
+When connecting to a PostgreSQL service, ClickHouse needs to know the name of the PostgreSQL schema and database you want to access. By default these settings are set to the public schema in the ``defaultdb``. However, you can update these values by following next steps.
+
+.. note::
+
+    Currently the configurations can be set only with the help of CLI command :ref:`avn service integration-update <avn service integration-update>`.
+
+
+1. Get *the service integration id* by requesting the full list of integrations. Replace ``CLICKHOUSE_SERVICE_NAME`` and ``PG_SERVICE_NAME`` with the names of your services:
+
+.. code::
+
+    avn service integration-list CLICKHOUSE_SERVICE_NAME | grep PG_SERVICE_NAME
+
+2. Update the configuration settings using the service integration id retrieved in the previous step and your integration settings. Replace ``SERVICE_INTEGRATION_ID``, ``PG_DATABASE`` and ``PG_SCHEMA`` with your values:
```

Comment:
Is this meaning that only one schema can be attached between CH and PG? What if I need to attach multiple schemas?

Reply:
databases is an array, so it accepts multiple items. 
Or, you mean more than one schema per same database? This should work - but this will create two different databases in CH, and not combining schemas. Will it make sense from PG perspective?
`"databases":[{"database":"PG_DATABASE","schema":"PG_SCHEMA1"}, {"database":"PG_DATABASE","schema":"PG_SCHEMA2"},]`

Reply:
I corrected the description to mention that you can have more than one

Reply:
Then it's fine, since we basically create the integration with the avnadmin user.
I believe, for external DBS, they'll likely create an integration per user so it should be fine too